README.md

```diff
@@ -153,18 +153,25 @@ Short description.
 
   App that calculates the climate impact of your daily choices by connecting to apps and services you already use: your personal and automated carbon tracker.
 
+- [Wren](https://projectwren.com) | United States | 2019
+
+  Wren is a service that offers a monthly subscription in order to offset your carbon footprint. Wren helps you calculate your footprint,
+  fund offset projects and track these offsets. 
+
 ## More Resources
 
 - [Clean Tech Finland](http://www.cleantechfinland.com/web/cleantech) - Hub of Finnish clean tech companies.
 - [r/ClimateActionPlan](https://www.reddit.com/r/ClimateActionPlan/) - A subreddit for giving people hope when facing climate change.
 - [Norrsken Foundation](https://www.norrskenfoundation.org/) - Swedish foundation which supports and invests in organizations that have a positive impact on society.
 - [Mattermore.io](https://www.mattermore.io/) - Newsletter highlighting companies using AI & data science to reverse climate change; useful for discovering new initiatives/work opportunities, delivered every week.
+- [Climate Choice](https://climatechoice.co/) - A website that contains actionable guides on reducing climate change, as well as educational information about climate change.
```

Comment:
Really cool resource! I think it fits better in the `Social Impact` category though, as there are similar companies there, just add in the description that it's a non-profit organization.

Reply:
Done.